Resources/views/privacycookie.html.twig

```diff
@@ -0,0 +1,41 @@
+<div class="privacy-cookie-banner" id="privacy-cookie-banner">
+    <div class="col-md-1"></div>
+    <div class="col-md-7 privacy-cookie-banner__message">
+        <p>{% if bannerCaption is empty %}
+               {{ 'Cookies help us create a good experience. By using our website, you agree to our use of cookies.'|trans }}
+           {% else %}
+               {{ bannerCaption }}
+           {% endif %}</p>
+    </div>
+    <div class="col-md-4 privacy-cookie-banner__action">
+        <div class="attribute-link pull-left">
+            <a class="btn" id="privacy-cookie-banner__privacy-accept" href="javascript:void(0)" title="{{ 'Agree'|trans }}">
```

Comment:
Of course, you can keep both classes: `privacy-cookie-banner__btn--accept` and `btn`. The `btn` class will be the context for the button module from Bootstrap, while the `privacy-cookie-banner__btn--accept` will have the context of privacy cookie banner. Modularity and keeping correct contexts.